File: MantenimientoDaxDirectorio/MantenimientoDirectorio/Emp.cs
using System;
using System.Data;
using DattCom;

namespace directMnt
{
	public class Emp
	{
		public static string PathImagenes;
		public static bool rol;
		public static bool OrdenaPorApellidos;  // esta en emp.Acumhis
		public static void CargarValores()
		{
			DataTable dt = datosEmpresa.leeParametrosEmp("EMP_PathImagenes,PAR_AcumHis");
			if (dt.Rows.Count > 0)
			{
				OrdenaPorApellidos = Convert.ToBoolean(dt.Rows[0]["PAR_AcumHis"]);
				PathImagenes = dt.Rows[0]["EMP_PathImagenes"].ToString();
			}
			dt.Dispose();
			try
			{
				if (datosEmpresa.auto.Length > 8)
					rol = datosEmpresa.auto.Substring(7, 1) == "1";
			}
			catch
			{
			}

		}

	}
}

File: MantenimientoDaxDirectorio/MantenimientoDirectorio/Emp.vb
Imports DattCom
Public Class Emp
    Public Shared PathImagenes As String
    Public Shared rol As Boolean
    Public Shared OrdenaPorApellidos As Boolean  ' esta en emp.Acumhis
    Public Shared Sub CargarValores()
        Dim dt As DataTable = DattCom.datosEmpresa.leeParametrosEmp("EMP_PathImagenes,PAR_AcumHis")
        If dt.Rows.Count > 0 Then
            OrdenaPorApellidos = Convert.ToBoolean(dt.Rows(0)("PAR_AcumHis"))
            PathImagenes = dt.Rows(0)("EMP_PathImagenes").ToString()
        End If
        dt.Dispose()
        Try
            If (datosEmpresa.auto.Length > 8) Then rol = (datosEmpresa.auto.Substring(7, 1) = "1")
        Catch
        End Try

    End Sub

End Class
